CMakeLists.txt

```diff
@@ -64,14 +64,21 @@ endif()
 include_directories(include ${PROJECT_BINARY_DIR}/include)
 
 configure_file(include/Config.in include/Config.h)
-configure_file(cmake/VecCoreConfig.cmake.in cmake/VecCoreConfig.cmake @ONLY)
+
+include(GNUInstallDirs)
```

Comment:
I would like to avoid including this if possible.

Reply:
Sure, do you think the user should be able to set the includedir and cmakedir, or can these always be `<CMAKE_INSTALL_PREFIX>/include` (always plus `VecCore`) and `<CMAKE_INSTALL_PREFIX>/share` (modulo comments below for Windows)?

Reply:
It should be enough to only set the prefix. I would say that setting directories individually is not usual.

Reply:
O.k, the update has removed `GNUInstallDirs` and provided non-cached variables for the include/cmake install locations relative to `CMAKE_INSTALL_PREFIX`.